chicago/templates/chicago/about.html

```diff
@@ -7,18 +7,24 @@
         <br class="non-mobile-only"/>
         <h1 id='about'>About</h1>
 
-        <p><strong>Chicago Councilmatic</strong> tracks all things related to Chicago City Council: the <a href='/search/' target="_blank">legislation</a> introduced and passed, its various <a href='/committees/' target="_blank">committees</a> and the <a href='/events/' target="_blank">meetings</a> they hold, and the <a href='/council-members/' target="_blank">aldermen</a> themselves.</p>
+        <p><strong>Miami Commissionmatic</strong> tracks all things related to Miami-Dade County Commission: the <a
+                href='/search/' target="_blank">legislation</a> introduced and passed, its various <a
+                href='/committees/' target="_blank">committees</a> and the <a href='/events/'
+                                                                              target="_blank">meetings</a> they hold,
+            and the <a href='/council-members/' target="_blank">aldermen</a> themselves.</p>
```

Comment:
Changes need to be made by hand. There are too many specific details to just grep our way through, in my opinion. 

> On Mar 22, 2016, at 10:06 AM, Ernie Hsiung <notifications@github.com> wrote:
> 
> In chicago/templates/chicago/about.html:
> 
> > @@ -7,18 +7,24 @@
> >          <br class="non-mobile-only"/>
> >          <h1 id='about'>About</h1>
> >  
> > -        <p><strong>Chicago Councilmatic</strong> tracks all things related to Chicago City Council: the <a href='/search/' target="_blank">legislation</a> introduced and passed, its various <a href='/committees/' target="_blank">committees</a> and the <a href='/events/' target="_blank">meetings</a> they hold, and the <a href='/council-members/' target="_blank">aldermen</a> themselves.</p>
> > +        <p><strong>Miami Commissionmatic</strong> tracks all things related to Miami-Dade County Commission: the <a
> > +                href='/search/' target="_blank">legislation</a> introduced and passed, its various <a
> > +                href='/committees/' target="_blank">committees</a> and the <a href='/events/'
> > +                                                                              target="_blank">meetings</a> they hold,
> > +            and the <a href='/council-members/' target="_blank">aldermen</a> themselves.</p>
> "Aldermen" is a Chicago specific term - they're called Commissioners here.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>